server/routes/adminDonor.ts

```diff
@@ -1,9 +1,9 @@
 import { Router } from 'express';
-import { deleteDonorById } from '../controllers';
+import { deleteDonorById, getDonorCampaigns } from '../controllers';
 import { authUser, authAdmin } from '../middlewares';
 
 const adminDonorRouter = Router();
 
 adminDonorRouter.route('/donor/:id').delete(authUser, authAdmin, deleteDonorById);
-
+adminDonorRouter.get('/donor/campagins/:id', authUser, authAdmin, getDonorCampaigns);
```

Comment:
what if you use ```/donor/:id/campagins``` instead ```/donor/campagins/:id```, isn't that more clearly and readable?

Reply:
yes i agree